app/services/windows.ts

```diff
@@ -203,6 +203,20 @@ export class WindowsService extends StatefulService<IWindowsState> {
     // This prevents "snapping" behavior when navigating settings
     if (options.componentName !== this.state.child.componentName) options.center = true;
 
+    // Override `options.size` when what is passed in is bigger than the current display
+    if (options.size) {
+      const { screen } = require('electron');
+      const { width: screenWidth, height: screenHeight } = screen.getPrimaryDisplay().workAreaSize;
+      const SCREEN_PERCENT = 0.75;
```

Comment:
Is the UX bad if this is `1.0`?  I'm hesitant because we're already making the windows smaller than they were designed in this case.  Making them even smaller makes me nervous.

Reply:
Yeah, with no downscaling you get the main window's titlebar and then right below the child window's titlebar and content spawning the entire display. The number I chose it's by no means scientific but it's the best one I found for keeping a "center/dialog"-like look.

Reply:
Got it, makes sense.